frontend/src/components/Tree/Tree.jsx

```diff
@@ -92,7 +102,22 @@ function renderTree(dimensions, jsonData, svgRef, onNodeClick) {
     .attr("text-anchor", "middle")
     .attr("font-size", 18)
     .attr("y", -15)
-    .text((node) => node.data.name);
+    .attr(
+      "transform",
+      (d) => `
+        rotate(${(d.x * 180) / Math.PI - 90}) 
+        translate(${d.y},0) 
+        rotate(${d.x >= Math.PI ? 180 : 0})
+      `
+    )
+    .attr("dy", "0.90em")
+    .attr("dx", "0.0em")
+    // eslint-disable-next-line
```

Comment:
Why do you have these? It seems to work fine without them on my side.

Reply:
That's strange; it wasn't working for me before but now it is working...pushing the version without the disable